R/provider-databricks.R

```diff
@@ -41,7 +53,7 @@ chat_databricks <- function(workspace = databricks_workspace(),
     base_url = workspace,
     model = model,
     extra_args = api_args,
-    api_key = token
+    api_key = token %||% ""
```

Comment:
I find this a bit confusing; I think it makes things a little simpler to reason about if we consistently use `NULL` to mean "use the default".

Reply:
I'd have to change the OpenAI provider for that, it doesn't currently accept a `NULL` `api_key`. Maybe I should define a nullable field for `token` on this provider instead?

Reply:
This is what I ended up doing. We no longer equivocate "API key" and "Databricks bearer token". That pattern is used in the Azure provider, too, so things are also more consistent now.